tsconfig.eslint.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "./tsconfigbase",
-  "include": ["packages/**/*", "app/**/*"],
+  "include": ["packages/**/*", "app/**/*", "ui-tests"],
```

Comment:
is this going to make things even worse re OoM?

Reply:
Probably yes, unfortunately I haven't been able to run ESLint on this repo for a while now... (still hitting #194 almost consistently)